### DISCUSSION TITLE
采矿镭射枪bug

### DISCUSSION THREAD

**Discussion by zgybd** [IndustrialCraftMC/IndustrialCraft2#13] [【漏洞反馈】Bugs Report]
1.超级热线模式 射击石头/深板岩/深板岩原石无限掉落
2. 爆破模式 无效果

**Comment by AbbottMc** [comment]
镭射枪已经移植完成，我不是很理解你说的爆破模式下无效果是啥意思。
我这边是都可以成功爆破的，就是远距离的时候看不到爆炸的粒子效果，这个大抵是实体的配置问题，后续会调整

**Reply by zgybd** [reply]
试了一下，超平坦爆破模式没效果，有爆炸动画。
